bin/swagger-repo.js

```diff
@@ -43,6 +43,7 @@ program
   .description('Builds the static assets and puts it ')
   .option('-b, --basedir <relpath>', 'The output file')
   .option('-o, --outdir <dirname>', 'The output directory, web_deploy by default')
+  .option('-s, --skipCodeSamples <skipCodeSamples>', 'Don\'t include the code samples')
```

Comment:
I think it's a boolean flag it doesn't require an argument `<skipCodeSamples>`. Could you remove it?

```suggestion
  .option('-s, --skipCodeSamples', 'Don\'t include the code samples')
```

Reply:
HI @RomanGotsiy 

I have removed the argument as requested.